static/app/views/settings/organizationAuthTokens/authTokenRow.spec.tsx

```diff
@@ -0,0 +1,156 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {
+  render,
+  renderGlobalModal,
+  screen,
+  userEvent,
+} from 'sentry-test/reactTestingLibrary';
+import {textWithMarkupMatcher} from 'sentry-test/utils';
+
+import OrganizationsStore from 'sentry/stores/organizationsStore';
+import {OrgAuthToken} from 'sentry/types';
+import {OrganizationAuthTokensAuthTokenRow} from 'sentry/views/settings/organizationAuthTokens/authTokenRow';
+
+describe('OrganizationAuthTokensAuthTokenRow', function () {
+  const {organization, router} = initializeOrg();
+
+  const revokeToken = jest.fn();
+  const token: OrgAuthToken = {
+    id: '1',
+    name: 'My Token',
+    tokenLastCharacters: 'XYZ1',
+    dateCreated: new Date('2023-01-01T00:00:00.000Z'),
+    scopes: ['org:read'],
+  };
+
+  const defaultProps = {
+    organization,
+    isRevoking: false,
+    token,
+    revokeToken,
+    projectLastUsed: undefined,
+    router,
+    location: router.location,
+    params: {orgId: organization.slug},
+    routes: router.routes,
+    route: {},
+    routeParams: router.params,
+  };
+
+  beforeEach(function () {
+    OrganizationsStore.addOrReplace(organization);
+  });
+
+  afterEach(function () {
+    MockApiClient.clearMockResponses();
+  });
+
+  it('shows token without last used information', async function () {
+    render(<OrganizationAuthTokensAuthTokenRow {...defaultProps} />);
+
+    expect(screen.getByLabelText('Token preview')).toHaveTextContent(
+      'sntrys_************XYZ1'
+    );
+    await screen.findByText('Never used');
+    await screen.findByText('My Token');
```

Comment:
```suggestion
    screen.getByText('My Token');
```

we probably only need to use `await` 1x

Reply:
do we actually need to be `async` here?

Reply:
yeah, true - I was moving stuff around a bit here, I'll adjust this!